sdk/storage/azure-storage-common/inc/azure/storage/common/constants.hpp

```diff
@@ -23,6 +23,10 @@ namespace Azure { namespace Storage {
     constexpr static const char* HttpHeaderContentType = "content-type";
     constexpr static const char* DefaultSasVersion = "2020-02-10";
 
+    // Error Codes.
+    constexpr static const char* ParentNotFound = "ParentNotFound";
+    constexpr static const char* ResourceNotFound = "ResourceNotFound";
```

Comment:
Why are these two so special that they should go in common `constants.hpp` rather than constants in each service?

Reply:
Those two might be used by other services. There is no specific resource related keyword.

Reply:
For example, they're used by which and which service?

Reply:
I don't have evidence it is never to be used in other services, now or in the future.

Reply:
If you don't have evidence, is it reasonable that it won't be used by multiple services?

Because if the other way around, we can add anything to common package.

Reply:
Moved to file service.